pages/purchase.tsx

```diff
@@ -0,0 +1,99 @@
+import Image from "next/image";
+import Head from "next/head";
+import Link from "next/link";
+import React from "react";
+import { ToggleHeader } from "../components/toggleHeader";
+import { PurchaseButtons } from "../components/purchase/PurchaseButtons";
+import { Container } from "react-bootstrap";
+import { useTranslation } from "next-i18next";
+import { serverSideTranslations } from "next-i18next/serverSideTranslations";
+import BookImage from "../public/front-cover-sm-424x600 1.svg";
+import SampleBookPageImage from "../public/sample book page.svg";
+import TranslationIcon from "../public/translationIcon.svg";
+import Footer from "../components/footer";
+
+export async function getStaticProps({ locale }: any) {
+    return {
+        props: {
+            ...(await serverSideTranslations(locale, ["purchase", "common"])),
+            // Will be passed to the page component as props
+            // About used in content, common used in header
+        },
+    };
+}
+
+const Purchase: React.FC = () => {
+    const { t } = useTranslation("purchase");
+
+    //TODO.localization: Japan prayer guide
+    return (
+        <div>
+            <Head>
+                <title>{t("title")}</title>
+                <meta name="description" content="Japan prayer guide" />
+                <meta name="viewport" content="width=device-width, initial-scale=1" />
+                <link rel="icon" href="/favicon.ico" />
+            </Head>
+            <ToggleHeader />
+            <main id="purchase">
+                <div className="book-description">
```

Comment:
I'm happy to leave this as is, just wanted to hear the team's thoughts though - 
do you think it would hurt maintainability if we didn't switch this over to the bootstrap style? using containers and bootstrap classes for the bulk of the styling etc

I _think_ it'll be okay? But let's be in agreement on this

Reply:
Is the bootstrap style using the utility styling?  If so, I found it much more limiting than using your own classes to define much more specific properties

Reply:
Yeah bootstrap would involve using the React-Boostrap components like <Container> or <Row> etc and then using the specific built in classes to do the styling. TBH it was hard to get used to but once I did it does become more consistent across the board.
Anyway I think its fine, but if it somehow becomes and issue later we can always revisit and replace the custom styling with bootstrap

Reply:
yeah the main thing is consistency - doing everything by yourself is always going to be less limiting, the benefit we get from bootstrap is the pre-built components & styling that makes is quick to pick up and use, and also we know those components will work with the sass variables, so in theory once styling is confirmed we can update the variables and know the components & classes will still all work together. 
As we're all okay with not following it in this case, i'll go ahead and approve